xmppserver/src/main/java/org/jivesoftware/openfire/update/AvailablePlugin.java

```diff
@@ -141,15 +139,7 @@ public static AvailablePlugin getInstance( Element plugin )
             minJavaVersion = new JavaSpecVersion( minJavaVersionValue );
         }
 
-        String releaseDate = null;
-        final String releaseDateString = plugin.attributeValue("releaseDate");
-        if( releaseDateString!= null) {
-            try {
-                releaseDate = RELEASE_DATE_DISPLAY_FORMAT.format(RELEASE_DATE_FORMAT.parse(releaseDateString));
-            } catch (final ParseException e) {
-                Log.warn("Unexpected exception parsing release date: " + releaseDateString, e);
-            }
-        }
```

Comment:
I think this exists because there are some older plugins out there with non-ISO release dates